client/src/app/tabs/dmn/diagram.dmn

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<definitions xmlns="http://www.omg.org/spec/DMN/20151101/dmn.xsd" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/1.0" id="Definitions_{{ ID }}" name="DRD" namespace="http://camunda.org/schema/1.0/dmn">
```

Comment:
Any reason why we remove the `{{ ID }}` placeholder, which got filled [later on](https://github.com/camunda/camunda-modeler/blob/master/client/src/app/TabsProvider.js#L217)?

Reply:
Restored the placeholder + added tests via https://github.com/camunda/camunda-modeler/pull/1186/commits/3335b042735c6e7645e05d38bd5e4a72155497fc.